content/systems/filecoin_nodes/repository/ipldstore/_index.md

```diff
@@ -12,7 +12,16 @@ dashboardTests: 0
 
 {{<embed src="../../../../libraries/ipld/ipld.id" lang="go" >}}
 
-IPLD is a set of libraries which allow for the interoperability of content-addressed data structures across different distributed systems. It provides a fundamental 'common language' to primitive cryptographic hashing, enabling data to be verifiably referenced and retrieved between two independent protocols. For example, a user can reference a git commit in a blockchain transaction to create an immutable copy and timestamp, or a data from a DHT can be referenced and linked to in a smart contract. 
+The InterPlanetary Linked Data (IPLD) is a set of libraries which allow for the interoperability of content-addressed data structures across different distributed systems an protocols. It provides a fundamental 'common language' to primitive cryptographic hashing, enabling data to be verifiably referenced and retrieved between two independent protocols. For example, a user can reference an IPFS directory in an Ethereum transaction or smart contract.
```

Comment:
Cut “The” from the beginning of the sentence.

s/an protocols/and protocols/

s/enabling data/enabling data structures/